src/main/java/com/google/sps/data/FormattedProject.java

```diff
@@ -1,17 +1,19 @@
 package com.google.sps.data;
 
 public final class FormattedProject {
-	private final String partner;
+	private final String collaborator;
+	private final String cname;
 	private final String projectid;
 	private final String title;
 	private final boolean bothActive;
 	private final String html;
 	private final String css;
 	private final String js;
 
-	public FormattedProject(String partner, String projectid, String title, 
+	public FormattedProject(String collaborator, String cname, String projectid, String title, 
```

Comment:
What is cname? can you make the name clearer? like collaboratorName if that's what this is